easydock/run_dock.py

```diff
@@ -213,6 +209,16 @@ def main():
                 del args_dict[arg]
             args.__dict__.update(args_dict)
 
+        has_finished_protonation = check_db_status(args.output, ['smi_protonated', 'source_mol_block_protonated']) and args.protonation
+        has_finished_docking = check_db_status(args.output, ['dock_time'])
+        if  has_finished_protonation or has_finished_docking:
```

Comment:
I suggest to change variable names once more, sorry for this. It seems more logical names would be `has_started...`, because we cannot say whether this stage was finished or not by this function output.